fuzzers/005-tilegrid/Makefile

```diff
@@ -96,6 +97,9 @@ cfg_int/build/segbits_tilegrid.tdb: build/basicdb/tilegrid.json
 orphan_int_column/build/segbits_tilegrid.tdb: build/basicdb/tilegrid.json
 	cd orphan_int_column && $(MAKE)
 
+clk_hrow/build/segbits_tilegrid.tdb: build/basicdb/tilegrid.json
```

Comment:
I feel like all of these rules could be generalized with a pattern. I know it's not related to the change, is there an issue for makefile clean up or replacing make? I'm happy to open one.

Reply:
Ya, go ahead and file an issue.  My make foo isn't that strong, and it's "only" 4 lines per.